src/api/applicant/content-types/applicant/lifecycles.js

```diff
@@ -1,14 +1,54 @@
+const _ = require('lodash');
+const fetch = require('node-fetch');
 const { createModuleResolutionCache } = require("typescript");
+const { Webhook, MessageBuilder } = require('discord-webhook-node');
+// create new Webhook
+const hook = new Webhook(
+  `${process.env.OPEN_POSITIONS_DISCORD_WEBHOOK}`
+);
+
+hook.setUsername("New Applicant Notifier")
+.setAvatar('https://avatars.githubusercontent.com/u/53379976?s=200&v=4');
+
+const formatJSON = (jsonData) => {
+  return `\`\`\`JSON\n${JSON.stringify(jsonData, null, 1)}\n\`\`\``;
```

Comment:
What are all the escaped slashes for?

Reply:
I think they're there so that the backticks can show up.
I think I could replace it with
```suggestion
  return `"```JSON\n" + `${JSON.stringify(jsonData, null, 1)}` + "\n```"`;
```